velox/dwio/parquet/writer/Writer.h

```diff
@@ -108,6 +109,8 @@ struct WriterOptions : public dwio::common::WriterOptions {
   /// Timestamp time zone for Parquet write through Arrow bridge.
   std::optional<std::string> parquetWriteTimestampTimeZone;
   bool writeInt96AsTimestamp = false;
+  // Default Parquet datapage version is V2.
+  std::optional<std::string> parquetDataPageVersion = "V2";
```

Comment:
Do not change the default version from V1 to V2: https://github.com/apache/arrow/blob/b8eae609a3e27ee6a1748183d7cf4785dae55b40/cpp/src/parquet/properties.h#L237C11-L237C29

Presto default should be set inside Prestissimo not in Velox.  Just leave it as `nullopt` here.

Reply:
Mistakenly believed we wanted to establish V2 as default. Fixed, thanks.